### DISCUSSION TITLE
SNMP on agents?

### DISCUSSION THREAD

**Discussion by xme** [Security-Onion-Solutions/securityonion#4200] [Unsupported Versions]
How can I reconfigure the firewall on an agent to accept SNMP traffic? (UDP/161).
I see the so-firewall command on the manager but does it work for agents too?

**Comment by cm-ops** [comment]
Do you mean the other Security Onion nodes in your grid? If so, see https://docs.securityonion.net/en/latest/firewall.html?highlight=Managing#managing on how to manage firewall rules for all devices in the grid.
